src/kibana/components/index_patterns/index_patterns.js

```diff
@@ -24,11 +25,17 @@ define(function (require) {
       return (new IndexPattern(id)).init();
     };
     
-    self.refresh = function() {
+    self.refreshNetworkIndex = function() {
       return self.get(DEFAULT_INDEX_PATTERN).then(function(indexPattern){
          indexPattern.refreshFields();
       });
-   };
+    };
+
+    self.refreshEventsIndex = function() {
+      return self.get(EVENTS_INDEX_PATTERN).then(function(indexPattern){
+         indexPattern.refreshFields();
+      });
+    };
```

Comment:
Is this something we could contribute back to the original kibana code? Obviously this is something that annoys many. 

What would the changes (not for this release) be that would make this useful for other users in the community?